src/Uno.UI/UI/Xaml/Data/BindingHelper.cs

```diff
@@ -8,9 +8,11 @@ namespace Uno.UI.Xaml
 	public static class BindingHelper
 	{
 		public static Binding SetBindingXBindProvider(Binding binding, object compiledSource, Func<object, object> xBindSelector, string[] propertyPaths = null)
-		{
-			binding.SetBindingXBindProvider(compiledSource, xBindSelector, propertyPaths);
+			=> SetBindingXBindProvider(binding, compiledSource, xBindSelector, null, propertyPaths);
 
+		public static Binding SetBindingXBindProvider(Binding binding, object compiledSource, Func<object, object> xBindSelector, Action<object, object> xBindBack, string[] propertyPaths = null)
+		{
+			binding.SetBindingXBindProvider(compiledSource, xBindSelector, xBindBack, propertyPaths);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'binding' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5087470)